packages/cli/src/bin/program.ts

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import chalk from 'chalk';
+import program from 'commander';
+import { Logger } from 'zos-lib';
+import commands from '../commands';
+import registerErrorHandler from './errors';
+
+require('./options');
+
+// TS-TODO: Decide whether this should be an interface or a type
+interface Command {
```

Comment:
Hmmm... We have a `Command` interface for our commands and a `Command` interface for commander. Maybe, we should 1) Rename this interface to something like `ZosCommand` and when importing commander's command, do it in a way that the type needs to be referenced with the namespace, like this: `commander.Command`

Reply:
I would prefer to import change our custom `Command` interface name rather to use `commander.Command`. Thx!